packaging/services/ovirt-engine/ovirt-engine.conf.in

```diff
@@ -691,7 +691,8 @@ SYSLOG_HANDLER_ENCODING=
 
 #
 # Formatter pattern to format the message to be sent to remote syslog
-SYSLOG_HANDLER_FORMATTER_PATTERN=
+# https://github.com/wildfly/wildfly/blob/main/docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Logging_Formatters.adoc#pattern-formatter
+SYSLOG_HANDLER_FORMATTER_PATTERN="%-5p [%c] (%t) %s%E%n"
```

Comment:
What %E stands for ?  I cannot find under the link above (there 'e' though)

Reply:
Hmm, that's a good point. I've just copied the value we are using for engine.log:

https://github.com/oVirt/ovirt-engine/blob/master/packaging/services/ovirt-engine/ovirt-engine.xml.in#L149

So it probably works due to some backward compatibility coincidence, but I'm going to change to comply with the spec.